deps/llvm-ver.make

```diff
@@ -1,3 +1,7 @@
+ifeq ($(USE_SYSTEM_LLVM), 1)
+LLVM_VER := $(shell $(LLVM_CONFIG_HOST) --version)
```

Comment:
yes, we shouldn't have the parsing of this file depend upon the result of a shell call. any errors in configuration then just get messy (e.g. if LLVM_CONFIG isn't found, then LLVM_VER becomes the empty string and all of deps breaks)